mautic/mautic
# Email address with two dots mark as invalid

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | ✅ 
| New feature/enhancement? (use the a.x branch)      | 
| Deprecations?                          | 
| BC breaks? (use the c.x branch)        | 
| Automated tests included?              | 
| Related user documentation PR URL      | 
| Related developer documentation PR URL | 
| Issue(s) addressed                     | 



#### Description:

Times ago PR https://github.com/mautic/mautic/pull/5078 added replace two dots with one dot. But I think we need make warning for customer If he want add these kind of email address, not correct it.



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Try add email address `something..test@test.com`
3. Before it was replaced with address  `something.test@test.com`
4. After PR should return validation error




<a href="https://gitpod.io/#https://github.com/mautic/mautic/pull/11258"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
